locales/en/translation.json

```diff
@@ -237,7 +237,7 @@
         "different": "You should use `!==` instead of `!=`",
         "custom-case": "There should be a `{{-goodOperator}}` where you wrote `{{-badOperator}}`"
       },
-      "declare-variable": "You haven't declared `{{-variable}}`.\nTry: `var {{-variable}};`",
+      "declare-variable": "You haven't declared `{{-variable}}`.\nTry coding it like this: `var {{-variable}};`",
```

Comment:
Same here—we know the variable name, so in this case the student can take the suggestion literally